.buildkite/pipeline.yml

```
@@ -0,0 +1,23 @@
+steps:
+  - label: "generator"
+    plugins:
+      - JuliaCI/julia#v1:
+          persist_depot_dirs: packages,artifacts,compiled
+          version: "1.7"
```

```suggestion
      - JuliaCI/merge-commit: ~
      - JuliaCI/julia#v1:
          persist_depot_dirs: packages,artifacts,compiled
          version: "1.7"
```

You'll also need to prepend this plugin to the plugin list for every step that you generate in your generator.

---

We'll also need to add it to the web UI.